src/main/charts/jira/values.yaml

```diff
@@ -1,48 +1,198 @@
+  # Valid values include:
+  # - 'postgres72'
+  # - 'mysql57'
+  # - 'mysql8'
+  # - 'oracle10g'
+  # - 'mssql'
+  # - 'postgresaurora96'
+  # Additional detail: 
+  # https://github.com/atlassian-labs/data-center-helm-charts/blob/master/docs/CONFIGURATION.md#databasetype
+  type: 
+  # -- The jdbc URL of the database
+  # If not specified, then it will need to be provided via the browser during initial startup. 
+  # Example URL's include:
+  # - 'jdbc:postgresql://<dbhost>:5432/<dbname>'
+  # - 'jdbc:mysql://<dbhost>/<dbname>'
+  # - 'jdbc:sqlserver://<dbhost>:1433;databaseName=<dbname>'
+  # - 'jdbc:oracle:thin:@<dbhost>:1521:<SID>'
+  # Additional detail: 
+  # https://github.com/atlassian-labs/data-center-helm-charts/blob/master/docs/CONFIGURATION.md#databasejdbcurl
+  url: 
+  # -- The Java class name of the JDBC driver to be used, e.g. 'org.postgresql.Driver'
   # If not specified, then it will need to be provided via the browser during initial startup.
-  url:
-  # -- The Java class name of the JDBC driver to be used, e.g. org.postgresql.Driver
-  # If not specified, then it will need to be provided via the browser during initial startup.
-  driver:
+  # Additional detail:
+  # https://github.com/atlassian-labs/data-center-helm-charts/blob/master/docs/CONFIGURATION.md#databasedriver
+  driver: 
   credentials:
     # -- The name of the Kubernetes Secret that contains the database login credentials.
-    # If specified, then the credentials will be automatically populated during Jira setup.
-    # Otherwise, they will need to be provided via the browser after initial startup.
-    secretName:
-    # -- The key in the Secret used to store the database login username
+    # If the secret is specified, then the credentials will be automatically utilised on 
+    # Jira startup. If the secret is not provided, then the credentials will need to be provided 
+    # via the browser during initial startup. Example of creating a Kubernetes secret below:
+    # 'kubectl create secret generic <secret-name> --from-literal=username=<username> \
+    # --from-literal=password=<password>'
+    # Additional detail:
+    # https://kubernetes.io/docs/concepts/configuration/secret/#opaque-secrets
+    secretName: 
+    # -- The key ('username') in the Secret used to store the database login username
     usernameSecretKey: username
-    # -- The key in the Secret used to store the database login password
+    # -- The key ('password') in the Secret used to store the database login password
     passwordSecretKey: password
 
+# - Volume configuration (REQUIRED FOR PRODUCTION) ######################################################################
+# By default, the charts will configure the local-home and shared-home as ephemeral volumes i.e. 
+# 'emptyDir: {}'. This is fine for evaluation purposes but for production deployments this is not 
+# ideal and so local-home and shared-home should both be configured appropriately. Additional detail:
```

Comment:
Possibly mention the potential for data-loss?

Reply:
Ive added a header to the file that calls this out